### PR TITLE
Feature: Support to run  Python code in Transformations

### LINKED ISSUE #4530: Run Python code to transform queries' response
## Description
Currently transforming query responses with **Transformation** only supports Javascript to manipulate or update the data.

### Proposed feature
Support to write **Python** code in **Transformation** along with Javascript

### PR DESCRIPTION
**Pull request by arpitnath**
Resolves #4530

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2x0knkx Add support for Python in transformations](https://app.clickup.com/t/2x0knkx)

File: frontend/src/_helpers/appUtils.js [hunk 1]
```diff
@@ -82,38 +82,126 @@ export function addToLocalStorage(object) {
...
+            # convert dictioanry to js object
+            return res
+          except Exception as e:
+            print(e)
+            return {"error": str(e)}
+            
+        exec_code
+    `);
+      const _data = JSON.stringify(payload);
+      result = execFunction(_data);
```
Comment (gondar00) [inline frontend/src/_helpers/appUtils.js:119]: I'm getting an error saying `execFunction` is not a function on this line

File: frontend/src/index.ejs [hunk 2]
```diff
@@ -32,6 +32,7 @@
       href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@400&display=swap"
       rel="stylesheet"
     />
+    <script src="https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js"></script>
```
Comment (gondar00) [inline frontend/src/index.ejs:35]: Is this required since we are loading it from local assets?

Comment (arpitnath) [reply frontend/src/index.ejs:35]: Yes, this adds the **loadPyodide** method to global object

File: frontend/src/_helpers/appUtils.js (hunk 1 above)
Comment (arpitnath) [reply frontend/src/_helpers/appUtils.js:119]: as it is scoped to the query, we have to return data or a primitive type

**Review by gondar00 — APPROVED** [review]

**Pull request merged**